modules/post-install-extend-existing-to-zone-type.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/aws-compute-edge-zone-tasks.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="post-install-existing-local-zone-subnet_{context}"]
+= Create network requirements in an existing VPC to use an AWS edge zone
```

Comment:
afaik "AWS edge zone" is not official, this is a term added by us to reference OCP deployment in LZ and/or WL
```suggestion
= Create network requirements in an existing VPC to use an AWS Wavelength or Local Zones
```

Reply:
Updated.